src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java

```diff
@@ -1182,6 +1182,7 @@ public void publish(String site, List<DeploymentItemTO> deploymentItems, String
 
                     Set<String> deployedCommits = new HashSet<String>();
                     Set<String> deployedPackages = new HashSet<String>();
+                    long start = System.currentTimeMillis();
```

Comment:
Add a check if the log level is `debug`, otherwise, don't compute this. The reason being: we don't log unless it's log-level debug.

Reply:
logger is crafter studio's wrapper that has embedded level check. However, it does not have exposed level check outside, so calculation will always happen.